scripts/quickstart

```diff
@@ -9,20 +9,24 @@ CERTS_DIR="${CONFIG_DIR}/certs"
 DOMAIN=openbalena.local
 
 usage() {
-  echo "usage: $0 [-h] [-p] [-d DOMAIN]"
+  echo "usage: $0 [-h] [-p] [-d DOMAIN] -U EMAIL -P PASSWORD"
   echo
   echo "  -p         patch hosts - patch the host /etc/hosts file"
-  echo "  -d DOMAIN  the domain name this deployment will run as, eg. example.com. Default is 'openbalena.local'"
+  echo "  -d DOMAIN  the domain name this deployment will run as, eg. example.com. Default is '${DOMAIN}'"
```

Comment:
If I invoke the command incorrectly like so:

```
./scripts/quickstart -d mydomain.com
```

it will cause an error which will print usage with:

```
...
-d DOMAIN  the domain name this deployment will run as, eg. example.com. Default is 'mydomain.com'
```

Reply:
good point... didn't think of that

Reply:
The joys of global state :)